src/middlewared/middlewared/alert/source/jails.py

```diff
@@ -0,0 +1,20 @@
+from middlewared.alert.base import AlertClass, AlertCategory, AlertLevel, OneShotAlertClass, Alert
+
+
+class PluginUpdateAlertClass(AlertClass, OneShotAlertClass):
+    deleted_automatically = False
+
+    category = AlertCategory.PLUGINS
+    level = AlertLevel.INFO
+    title = 'Plugin Update Available'
+    text = 'Plugin "%(name)s" update is available. Go to ' \
+           'Plugins->Installed->"%(name)s"->Update and apply the update'
```

Comment:
Corrected, but it is a mistake for the middleware to suggest where the user should go in the menus. That is a separate place to track and get out of sync when menus or buttons change. It would be better to be more vague here, leaving out the second sentence entirely.  The user can find the plugin update in the user interface by themselves.  Or if they can't, it's a separate issue we should fix there.
```suggestion
           'Plugins -> Installed -> "%(name)s"->Update to apply the update.'
```

Reply:
I'll omit the last sentence. Thank you